### PR TITLE
Make image for car and heating visible, not enough money function.

### PR DESCRIPTION
**Pull request by Nolle10**
In this pull request, we added so that the Game class, can see if the player has enough money to buy and complete a year with their current economy, if they do not have the means the game will end.

Also added so that when you buy a car and a heater, they will be visible at the player house.

### CODE REVIEW

File: src/main/java/worldofzuul/Game.java [hunk 1]
```diff
@@ -347,21 +347,38 @@ public boolean nextYear(Command command) {
...
             // Step 8: Print to the player what happened(money, co2, sold energy, sales price)
             System.out.println("You are now in the year: " + gameYear);
-            // TODO: Print the rest of what happened
+            System.out.println("Your emission for this year is: " + player.getRecapEnergyEmission().get(gameYear-1));
+            System.out.println("Your total emission is: " + player.calculateEmission());
+            System.out.println("Your earned money on sold energy: " + soldEnergyPrice);
+            System.out.println("Your balance are:" + player.getPlayerEconomy());
             return false;
         } else {
-            System.out.printf("Please fulfill the required amount of energy, missing: %.2fkWh\n", house.getEnergyRequirement() - buildArea.getYearlyEnergyProduction());
```
Comment (mathiasgredal) [inline src/main/java/worldofzuul/Game.java:353]: We still need to print this if we found a solution

File: src/main/java/worldofzuul/presentation/NextYearController.java [hunk 2]
```diff
@@ -66,5 +62,6 @@ private void printYearlyRecap(){
         totalEmissionText.setText("Your total emission is: " + decimalFormat.format(Game.instance.getPlayer().calculateTotalEmission()) + " kg CO\u2082");
         earnedMoneyText.setText("Your earned money on sold energy: " + decimalFormat.format(Game.instance.getSoldEnergyPrice()) + " DKK");
         balanceText.setText("Your balance are: " + decimalFormat.format(Game.instance.getPlayer().getPlayerEconomy()) + " DKK");
+
```
Comment (mathiasgredal) [inline src/main/java/worldofzuul/presentation/NextYearController.java]: Why the blank space?

File: src/main/java/worldofzuul/presentation/RetailController.java [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        Shop foundShop = null;
+        for (int i = 0; i < Game.instance.getShops().size(); i++) {
+            if (Game.instance.getShops().get(i).getName().equals(finalShopName)) {
+                foundShop = Game.instance.getShops().get(i);
+            }
+        }
+        if (foundShop != null) {
+            Game.instance.buyItem(new Command(CommandWord.BUY, ((ImageView) mouseEvent.getSource()).getId()), foundShop);
+        }
+    }
```
Comment (mathiasgredal) [inline src/main/java/worldofzuul/presentation/RetailController.java]: We use the buy button now, so this can be deleted

**Comment by mathiasgredal** [comment]
We could also make some tooltips for the car and the heater

**Review by mathiasgredal — CHANGES_REQUESTED** [review]
Mostly good

**Comment by Nolle10** [comment]
Could is the keyword in this sentence, nah just kidding, i will look at it

**Review by mathiasgredal — APPROVED** [review]
LGTM

**Pull request merged**